bayesian/api_v1.py

```diff
@@ -347,35 +347,42 @@ def get(external_request_id):
                 started_at = stack_result["task_result"]["_audit"]["started_at"]
                 finished_at = stack_result["task_result"]["_audit"]["ended_at"]
 
-
         if reco_result is not None and 'task_result' in reco_result:
-            if reco_result["task_result"] != None:
-                recommendation = reco_result['task_result']['recommendations']
+                if reco_result["task_result"] != None:
+                    recommendation = reco_result['task_result']['recommendations']
 
         # Populate sentiment score for packages in user's stack
         if stack_result is not None:
-            user_stack_deps = stack_result['task_result']['user_stack_info']['dependencies']
-            for dep in user_stack_deps:
-                if user_stack_sentiment_result is not None:
-                    dep['sentiment']['overall_score'] = user_stack_sentiment_result['task_result'][dep['name']]['score']
-                else:
-                    dep['sentiment'] = {}
+            user_stack_deps = stack_result.get('task_result').get('user_stack_info').get('dependencies',[])
```

Comment:
if default value is not specified, then we can use subscription method to fetch a value for given key. For example, here we can say `stack_result['task_result']['user_stack_info'].get('dependencies', [])` as this is more readable.